mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java

```diff
@@ -681,7 +681,6 @@ private void handleWebSocketUpgrade(Socket socket, BufferedSource source, Buffer
     } catch (IOException e) {
       webSocket.failWebSocket(e, null);
     } finally {
-      closeQuietly(sink);
```

Comment:
It's hard to tell from jumping around here in the PR, but doesn't `processConnection` also close both the `Sink` and the `Source`?

Reply:
Oooh, good catch. I just removed those calls; they aren’t useful.

Reply:
(Well, useful only in that they could cause segments to be recycled that wouldn’t be otherwise. Not worth writing more code for, especially since in normal operations the buffers will be empty.)